python/src/lib.rs

```diff
@@ -118,12 +120,13 @@ struct RawDeltaTableMetaData {
 #[pymethods]
 impl RawDeltaTable {
     #[new]
-    #[pyo3(signature = (table_uri, version = None, storage_options = None, without_files = false))]
+    #[pyo3(signature = (table_uri, version = None, storage_options = None, without_files = false, init = true))]
```

Comment:
Forgive my naivete with pyo3 signatures, but wouldn't it be:
```suggestion
    #[pyo3(signature = (table_uri, version = None, storage_options = None, without_files = False, init = True))]
```